### DISCUSSION TITLE
Sort Rank Breakdown by Rank

### DISCUSSION THREAD

**Discussion by Kacarott** [codewars/codewars.com#2904] [Ideas]
Currently the "Rank breakdown" in user profiles is sorted alphabetically. I think it makes a lot more sense for it to be sorted with highest rank first. (This way, I wont have to stare at my 8kyu rank in C every time I am in my profile)
![](https://user-images.githubusercontent.com/43819332/212699350-5519258d-71a6-4ba5-8dcd-7283c50e32ff.png)

**Comment by hobovsky** [comment]
Just for your information, POLYGLOT userscript does this.

**Comment by ejini6969** [comment]
Or maybe provide a sorting criteria same as that of kata search page :)

**Comment by kazk** [comment]
Deployed
